third_party/musl/crt/patch.diff

```diff
@@ -551,6 +551,96 @@ index 02cb1aa2..07e76f19 100644
  	} else if ((name&~4U)!=1 && name-_CS_POSIX_V6_ILP32_OFF32_CFLAGS>33U) {
  		errno = EINVAL;
  		return 0;
+diff --git a/src/conf/fpathconf.c b/src/conf/fpathconf.c
+index e6aca5cf..d7392ced 100644
+--- a/src/conf/fpathconf.c
++++ b/src/conf/fpathconf.c
+@@ -1,6 +1,7 @@
+ #include <unistd.h>
+ #include <limits.h>
+ #include <errno.h>
++#include <sys/stat.h>
+ 
+ long fpathconf(int fd, int name)
+ {
+@@ -31,5 +32,20 @@ long fpathconf(int fd, int name)
+ 		errno = EINVAL;
+ 		return -1;
+ 	}
++
++	// The maximum length of a filename in the directory path or
++	// fd that the process is allowed to create
++	if (name == _PC_NAME_MAX) {
```

Comment:
Is it easy in musl to move this logic to a function which both `pathconf` and `fpathconf` can use?

Reply:
I couldn't think of a way to reuse this, since one is pathname and the other is fd?